build-farm/platform-specific-configurations/mac.sh

```diff
@@ -68,6 +68,11 @@ else
   fi
 fi
 
+if [[ ("${VARIANT}" == "${BUILD_VARIANT_TEMURIN}" || "${VARIANT}" == "${BUILD_VARIANT_HOTSPOT}") && "$JAVA_FEATURE_VERSION" -ge 21 ]]; then
+  # Temurin/hotspot jdk-21+ uses "bundled" FreeType
+  export BUILD_ARGS="${BUILD_ARGS} --freetype-dir bundled"
```

Comment:
If we're doing this for mac, can I suggest limiting the use of "--skip-freetype" [here](https://github.com/adoptium/temurin-build/blob/da2408e4ea988090835f15f29cb170873cced045/.github/workflows/build.yml#L170) to jdk21+?

I mean, this code will override that code, so this isn't a functional improvement. It just prevents us having a "skip" followed by a "bundle", which could confuse developers reading the command.

Reply:
thanks Adam, i've updated that yml, it does not actually need to specify --skip-freetype as that determination is already done in prepareWorkspace.sh/build.sh